src/desktop/apps/artwork/components/partner/index.jade

```diff
@@ -23,6 +23,7 @@ if helpers.partner.isDisplayable(artwork.partner)
           = profile.bio
 
       .artwork-partner__metadata__actions
+         //- TODO BNMO: Update to hide if the artwork is acquireable
```

Comment:
This component is a little partner info widget on the artwork view:
![image](https://user-images.githubusercontent.com/2081340/45393382-aab0d580-b5f9-11e8-8cc2-ac7ae3b66085.png)

It only shows if the artwork's partner is a [gallery or an institution](https://github.com/artsy/force/blob/master/src/desktop/apps/artwork/components/partner/helpers.coffee#L5).

So, when we eventually want to suppress inquiry buttons for `acquireable` artworks, we'll have to update this. For now it's okay.